gsoc/cern.md

```diff
@@ -8,5 +8,5 @@ layout: default
 ## Project Proposals
 
 {:.table .table-hover .table-striped}
-{% for post in site.categories.gsoc %}{% if post.organization == "CERN" %} | [ {{ post.title }} ]( {{ post.url }} ) | {% endif %} 
+{% for page in site.pages %}{% if page.categories contains 'gsocproposal' and page.organization == "CERN" %} | [ {{ page.title }} ]( {{ page.url }} ) | {% endif %} 
```

Comment:
Yes it is. Look at https://github.com/HEP-SF/hep-sf.github.io/blob/master/_includes/navbar.ext#L20. This is one of the great features of collections. A great post IMO describing many of the advanced features of collections is http://ben.balter.com/2015/02/20/jekyll-collections/....